pages/about.md

```diff
@@ -85,6 +85,34 @@ The FMI development was partially funded within the ITEA2 project MODELISAR by
   </tr>
 </table>
 
+## Trademark Guidelines for the Use of the FMI Trademark
+
+### Preliminaries
+
+The FMI project and the Modelica Association (MA) are asking all members of the public and the FMI community to follow and respect the following usage guidelines for the use of the FMI trademark.
+The purpose of the trademark registration is to protect the use of the word-mark FMI in the protected application domains from any usage that is contrary to the mission and values of the trademark owner, the Modelica Association.  
+The purpose of this guideline is to make sure that any use of the FMI trademark agrees with these guidelines and strengthens the brand and value of the FMI standard.
+Further, the guidelines will try to make sure that no confusion or misinformation can arise from the use of the FMI trademark.
+At the same time, the Modelica Association wants to encourage as many commercial companies and open source software developers to support the FMI standard in a high quality of implementation and put as little administrative burden as possible.
+The MA encourages to mention the FMI standard with a proper reference. The main reference in any tool documentation should contain a pointer to the official FMI website http://www.fmi-standard.org.
+The Modelica Association expects all tool vendors that implement the FMI standard to participate in the cross-check protocol, especially those using FMI in their product name.
+
+### Trademark Usage Guidelines
+
+The use of the letters FMI in a product name or other relevant reference is allowed when
```

Comment:
Maybe the term "FMI" should be emphasized, e.g., by butting in in quotes are have it as _FMI_.

Reply:
Done (hopefully syntactically correct).